src/composables/Utilities.ts

```diff
@@ -28,6 +28,30 @@ export const Growth: Record<string, GrowthFunc> = {
   },
 };
 
+export class Util {
```

Comment:
Yep, let's use the `CoinDisplay`. If you need help refactoring/adapting, let me know.